src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
```

Comment:
You need use variables for the main values - checklist point 1
And don't use else after return statement - checklist point 3
🙂

Reply:
fixed